### PR TITLE
dev-php/pecl-mcrypt: bump to 1.0.3

### PR DESCRIPTION
**Pull request by hydrapolic**
Closes: https://bugs.gentoo.org/696620
Package-Manager: Portage-2.3.77, Repoman-2.3.17
Signed-off-by: Tomáš Mózes <hydrapolic@gmail.com>

### CODE REVIEW

File: dev-php/pecl-mcrypt/pecl-mcrypt-1.0.3.ebuild [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PHP_EXT_NAME="mcrypt"
+USE_PHP="php7-2 php7-3"
```
Comment (ConiKost) [inline dev-php/pecl-mcrypt/pecl-mcrypt-1.0.3.ebuild]: What abould ```php7-4```? Is it supported?

Comment (hydrapolic) [reply dev-php/pecl-mcrypt/pecl-mcrypt-1.0.3.ebuild]: Haven't tested yet

Comment (ConiKost) [reply dev-php/pecl-mcrypt/pecl-mcrypt-1.0.3.ebuild]: Can I help you with testing? I have here running php7.4, but I don't know how to test this module propably?

Comment (hydrapolic) [reply dev-php/pecl-mcrypt/pecl-mcrypt-1.0.3.ebuild]: Sure, that would help. Just run the tests, it should all pass.

FEATURES=test emerge dev-php/pecl-mcrypt

Comment (ConiKost) [reply dev-php/pecl-mcrypt/pecl-mcrypt-1.0.3.ebuild]: I've run the tests with ```php7-4``` and all tests passed.

```
=====================================================================
PHP         : /usr/lib64/php7.4/bin/php
PHP_SAPI    : cli
PHP_VERSION : 7.4.0RC3
ZEND_VERSION: 3.4.0-dev
PHP_OS      : Linux - Linux Bl4ckB0x 5.3.6-gentoo-r1 #1 SMP Sat Oct 12 20:30:46 CEST 2019 x86_64
INI actual  : /var/tmp/portage/dev-php/pecl-mcrypt-1.0.3/work/php7.4/tmp-php.ini
More .INIs  :
CWD         : /var/tmp/portage/dev-php/pecl-mcrypt-1.0.3/work/php7.4
Extra dirs  :
VALGRIND    : Not used
=====================================================================
TIME START 2019-10-20 12:05:36
=====================================================================
PASS Test for blowfish compatibility [tests/blowfish.phpt]
PASS Bug #35496 (Crash in mcrypt_generic()/mdecrypt_generic() without proper init). [tests/bug35496.phpt]
PASS Bug #37595 (mcrypt_generic calculates data length in wrong way) [tests/bug37595.phpt]
PASS Bug #41252 (Calling mcrypt_generic without first calling mcrypt_generic_init crashes) [tests/bug41252.phpt]
PASS Bug #43143 (Warning about empty IV with MCRYPT_MODE_ECB) [tests/bug43143.phpt]
PASS Bug #46010 (warnings incorrectly generated for iv in ecb mode) [tests/bug46010.phpt]
PASS Bug #49738 (calling mcrypt after mcrypt_generic_deinit crashes) [tests/bug49738.phpt]
PASS mcrypt_create_iv https://bugs.php.net/bug.php?id=55169 [tests/bug55169.phpt]
PASS Bug #70625: mcrypt_encrypt() : won't return data when no IV was specified under RC4 [tests/bug70625.phpt]
PASS Bug #8040 (MCRYPT_MODE_* do not seem to exist) [tests/bug8040.phpt]
PASS mcrypt_cbc [tests/mcrypt_cbc.phpt]
PASS Test mcrypt_cbc() function : basic functionality [tests/mcrypt_cbc_3des_decrypt.phpt]
PASS Test mcrypt_cbc() function : basic functionality [tests/mcrypt_cbc_3des_encrypt.phpt]
PASS mcrypt_cfb [tests/mcrypt_cfb.phpt]
PASS mcrypt_create_iv [tests/mcrypt_create_iv.phpt]
PASS mcrypt_decrypt [tests/mcrypt_decrypt.phpt]
PASS Test mcrypt_decrypt() function : basic functionality [tests/mcrypt_decrypt_3des_cbc.phpt]
PASS Test mcrypt_decrypt() function : basic functionality [tests/mcrypt_decrypt_3des_ecb.phpt]
PASS Test mcrypt_decrypt() function : error conditions [tests/mcrypt_decrypt_error.phpt]
PASS Test mcrypt_decrypt() function : usage variation [tests/mcrypt_decrypt_variation1.phpt]
PASS Test mcrypt_decrypt() function : usage variation [tests/mcrypt_decrypt_variation2.phpt]
PASS Test mcrypt_decrypt() function : usage variation [tests/mcrypt_decrypt_variation3.phpt]
PASS Test mcrypt_decrypt() function : usage variation [tests/mcrypt_decrypt_variation4.phpt]
PASS Test mcrypt_decrypt() function : usage variation [tests/mcrypt_decrypt_variation5.phpt]
PASS mcrypt_ecb [tests/mcrypt_ecb.phpt]
PASS Test mcrypt_cbc() function : basic functionality [tests/mcrypt_ecb_3des_decrypt.phpt]
PASS Test mcrypt_ecb() function : basic functionality [tests/mcrypt_ecb_3des_encrypt.phpt]
PASS mcrypt_enc_get_algorithms_name [tests/mcrypt_enc_get_algorithms_name.phpt]
PASS mcrypt_enc_get_block_size [tests/mcrypt_enc_get_block_size.phpt]
PASS mcrypt_enc_get_iv_size [tests/mcrypt_enc_get_iv_size.phpt]
PASS mcrypt_enc_get_key_size [tests/mcrypt_enc_get_key_size.phpt]
PASS mcrypt_enc_get_modes_name [tests/mcrypt_enc_get_mode_name.phpt]
PASS mcrypt_enc_get_supported_key_sizes [tests/mcrypt_enc_get_supported_key_sizes.phpt]
PASS mcrypt_enc_is_block_algorithm [tests/mcrypt_enc_is_block_algorithm.phpt]
PASS mcrypt_enc_is_block_algorithm_mode [tests/mcrypt_enc_is_block_algorithm_mode.phpt]
PASS mcrypt_enc_is_block_mode [tests/mcrypt_enc_is_block_mode.phpt]
PASS mcrypt_enc_self_test [tests/mcrypt_enc_self_test.phpt]
PASS Test mcrypt_encrypt() function : TripleDES functionality [tests/mcrypt_encrypt_3des_cbc.phpt]
PASS Test mcrypt_encrypt() function : TripleDES functionality [tests/mcrypt_encrypt_3des_ecb.phpt]
PASS Test mcrypt_encrypt() function : error conditions [tests/mcrypt_encrypt_error.phpt]
PASS Test mcrypt_encrypt() function : usage variation - different types for cipher [tests/mcrypt_encrypt_variation1.phpt]
PASS Test mcrypt_encrypt() function : usage variation [tests/mcrypt_encrypt_variation2.phpt]
PASS Test mcrypt_encrypt() function : usage variation [tests/mcrypt_encrypt_variation3.phpt]
PASS Test mcrypt_encrypt() function : usage variation [tests/mcrypt_encrypt_variation4.phpt]
PASS Test mcrypt_encrypt() function : usage variation [tests/mcrypt_encrypt_variation5.phpt]
PASS mcrypt filters [tests/mcrypt_filters.phpt]
PASS mcrypt_get_block_size [tests/mcrypt_get_block_size.phpt]
PASS mcrypt_get_cipher_name [tests/mcrypt_get_cipher_name.phpt]
PASS mcrypt_enc_get_iv_size [tests/mcrypt_get_iv_size.phpt]
PASS mcrypt_get_key_size [tests/mcrypt_get_key_size.phpt]
PASS mcrypt_list_algorithms [tests/mcrypt_list_algorithms.phpt]
PASS mcrypt_list_modes [tests/mcrypt_list_modes.phpt]
PASS mcrypt_module_get_algo_block_size [tests/mcrypt_module_get_algo_block_size.phpt]
PASS mcrypt_module_get_algo_key_size [tests/mcrypt_module_get_algo_key_size.phpt]
PASS mcrypt_module_get_supported_key_sizes [tests/mcrypt_module_get_supported_key_sizes.phpt]
PASS mcrypt_module_is_block_algorithm [tests/mcrypt_module_is_block_algorithm.phpt]
PASS mcrypt_module_is_block_algorithm_mode [tests/mcrypt_module_is_block_algorithm_mode.phpt]
PASS mcrypt_module_is_block_mode [tests/mcrypt_module_is_block_mode.phpt]
PASS mcrypt_module_open [tests/mcrypt_module_open.phpt]
PASS mcrypt_module_self_test [tests/mcrypt_module_self_test.phpt]
PASS mcrypt_ofb [tests/mcrypt_ofb.phpt]
PASS Test mcrypt_encrypt() function : AES functionality [tests/mcrypt_rijndael128_128BitKey.phpt]
PASS Test mcrypt_encrypt() function : AES functionality [tests/mcrypt_rijndael128_256BitKey.phpt]
=====================================================================
TIME END 2019-10-20 12:05:39

=====================================================================
TEST RESULT SUMMARY
---------------------------------------------------------------------
Exts skipped    :    0
Exts tested     :   54
---------------------------------------------------------------------

Number of tests :   63                63
Tests skipped   :    0 (  0.0%) --------
Tests warned    :    0 (  0.0%) (  0.0%)
Tests failed    :    0 (  0.0%) (  0.0%)
Tests passed    :   63 (100.0%) (100.0%)
---------------------------------------------------------------------
Time taken      :    3 seconds
=====================================================================
```

Comment (hydrapolic) [reply dev-php/pecl-mcrypt/pecl-mcrypt-1.0.3.ebuild]: Thanks for testing, I'll add php 7.4 too :)

Comment (ConiKost) [reply dev-php/pecl-mcrypt/pecl-mcrypt-1.0.3.ebuild]: Thanks :-)

Comment (hydrapolic) [reply dev-php/pecl-mcrypt/pecl-mcrypt-1.0.3.ebuild]: Thank you, updated the PR.

**Comment by juippis** [comment]
Thank you, both!

**Comment by hydrapolic** [comment]
Thanks @juippis and @ConiKost

**Pull request closed without merging**